keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,464 @@
+  * Exceptions only approach for a receipt to be added back to release cycle stage after the deadline
+  * Enhancement Team + Release Leads should have the final say on exceptions. The decision needs to be taken after considering the point of view of respective SIG leadership.
+* Conflicts can arise when two people update/modify the same artifact
+  * The team needs to have Git experience
+  * Only one person works on the artifacts at a time
+  * Communicate with each other when making changes
+  * Bots can eventually handle the manifest generation
+* Keeping KEP implementation history updated
+  * Requires intervention from a human and trust needs to be on the same person
+* Preventing enhancements to slip across release cycles
+* Ensuring enhancements don't permanently remain in the Beta stage
+* This KEP doesn't talk about KEP owners keeping enhancements issues up-to-date
+
+## Design Details
+
+
+### How does a Receipt look like?
+
+A Receipt has the following data stored at any point in time.
+
+```yaml
+title: Receipts process for tracking release enhancements
+kep-number: 2140
+owning-sig: sig-architecture
+path: sig-architecture/2140-receipts
+tracking-status: proposed
+stage: alpha
+```
+
+### Where are the Receipts stored?
+
+The Receipts will be stored in `releases` directory of the [k/enhancements][k-enhancements] repository. Each release will have a subdirectory and the structure would look like this.
+
+```
+tree releases
+releases
+└── v1.21
+    ├── accepted
+    │   └── 0001.yaml
+    ├── at-risk
+    │   └── 0002.yaml
+    ├── exception
+    │   └── 0003.yaml
+    ├── OWNERS
+    ├── proposed
+    │   └── 0004.yaml
+    └── removed
+        └── 0005.yaml
+```
+
+The `OWNERS` file contains the Enhancements Team for that specific Release cycle.
+
+### State Diagram
+
+![Receipts State Diagram](state-diagram.png)
+
+The flows shown in the state diagram are described in the sections below. Some of the flows described will have appropriate `kepctl` commands from the `Alpha` version itself. Other helper commands will be added and existing commands refined throughout the rollout process.
+### Initializing the Receipts directory for a Release
+
+In the initial rollout of this process, the release team will manually generate this structure, however, `kepctl` will provide a command to generate this structure:
+
+
+```
+$ kepctl create release v1.22
+==> 1.22 Release Created
+$ tree releases
+releases
+└── v1.22
+    ├── accepted
+    ├── at-risk
+    ├── exception
+    ├── OWNERS
+    ├── proposed
+    └── removed
+```
+
+### Proposing a KEP for a Release
+
+A KEP author or SIG Chair/Tech Lead opts-in to a given release by creating the receipt document ([described above](#where-are-the-receipts-stored)) in the `proposed`  directory for the specified release. The `latest-milestone` and `stage` should be updated in the associated `kep.yaml` file as well. This process can be done manually or with `kepctl`.
```

Comment:
If a KEP missed the previous release (and so is in the `removed` directory of the previous release's structure), would the PR to propose it for the upcoming release add a whole new receipt, or move the receipt from the `removed` directory of the previous?